code/modules/antagonists/bloodsucker/datum_bloodsucker.dm

```diff
@@ -200,9 +200,9 @@
 		var/mob/living/carbon/human/H = owner.current
 		var/datum/species/S = H.dna.species
 		// Make Changes
-		S.brutemod *= 0.5											//  <--------------------  Start small, but burn mod increases based on rank!
+		S.brutemod *= 0.8										//  <--------------------  Start small, but burn mod increases based on rank!
 		S.coldmod = 0
-		S.stunmod *= 0.25
+		S.stunmod *= 0.35
 		S.siemens_coeff *= 0.75 	//base electrocution coefficient  1
 		//S.heatmod += 0.5 			// Heat shouldn't affect. Only Fire.
 		//S.punchstunthreshold = 8	//damage at which punches from this race will stun  9
```

Comment:
Just noticed how it modifies the current species' coefficients instead of the physiology datum's, and with upmost certainty will be fucked up by species change.
Please use the physiology equivalents where possible (cause there is no variable for punches min / max threshold yet) instead.
Here is the [physiology datum file](https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/code/modules/mob/living/carbon/human/physiology.dm)
Also the var name:
https://github.com/Citadel-Station-13/Citadel-Station-13/blob/7e5e241bd2c72dd3e451344ee5e6aa9dca9bed41/code/modules/mob/living/carbon/human/human_defines.dm#L63

Also the fact these buffs are not undone on antag removal worries me a little but that's outside the scope of this PR.

Reply:
uhhh it should be undone on antag removal

Reply:
`	if(ishuman(owner.current))
		var/mob/living/carbon/human/H = owner.current
		H.set_species(H.dna.species.type)`